test/integration/features/addons/addon-registry-route.feature

```diff
@@ -57,17 +57,22 @@ so that user can able to push their container images directly to registry and de
      Then with up to "10" retries with wait period of "500ms" the "status code" of HTTP request to "/" of service "nginx" in namespace "myproject" is equal to "200"
       And with up to "10" retries with wait period of "500ms" the "body" of HTTP request to "/" of service "nginx" in namespace "myproject" contains "Welcome to nginx!"
 
-  Scenario: User restarts Minishift
+  Scenario: User can login to OpenShift integrated docker registry after Minishift restart
     Given Minishift has state "Running"
      When executing "minishift stop" succeeds
      Then executing "minishift start" succeeds
+      And Minishift should have state "Running"
+     Then with up to "20" retries with wait period of "3" seconds container name "registry_docker-registry" should be "running"
+      And with up to "60" retries with wait period of "1000ms" the "status code" of HTTP request to "/" of service "docker-registry" in namespace "default" is equal to "200"
+     When executing "docker login -u developer -p `oc whoami -t` docker-registry-default.$(minishift ip).nip.io" in host shell succeeds
+     Then stdout of host shell should contain "Login Succeeded"
 
-  Scenario: User can login to OpenShift integrated docker registry after restart minishift
+  Scenario: User can reapply registry-route addon and login to OpenShift integrated docker registry
```

Comment:
This also can be split into 2 scenarios. Scenario: User can reapply addon and Scenario: Login to OpenShift registry

Reply:
sure